src/pages/en/guides/integrations-guide.md

```diff
@@ -15,6 +15,8 @@ title: Using Integrations
 
 In this example, we will add the `@astrojs/react` integration to add React support to your Astro project. The process for adding any other framwork (Preact, Vue, Svelte or Solid.js) is almost identical and can be followed using the same steps outlined below.
 
+> Feeling adventurous? Astro recent launched an **experimental** `astro add` command to automate this process! If you'd like to try it, just run `astro add react`. That's it!
```

Comment:
```suggestion
> Feeling adventurous? Astro recent launched an **experimental** `astro add` command to automate this process! Instead of the steps below, you can just run `npx astro add react`. That's it! Skip down to [Automated Integration Setup](/en/guides/integrations-guide/#automatic-integration-setup) for more details.
```

Nit nit: I love this call-out, and the wording itself is perfect, but when I look at it in the preview, visually it doesn't scream "Stop!" to me and isn't separated enough from  "First, you will need to.." instructions. 

So, the above is just a suggestion you can take, leave or improve upon, but I would make sure to look at this in the deploy preview for confirmation that it's visually clear, as well. (You could even consider an emoji to start the quote block.)

Reply:
Appreciate the thoughts! I'll see if there's a way to make this stand out more ☺️

I do love your wording suggestions, though. Added those!